messaging/src/main/java/org/axonframework/messaging/MessageStream.java

```diff
@@ -57,6 +60,19 @@ static <M extends Message> MessageStream<M> fromIterable(@Nonnull Iterable<M> it
         return fromIterable(iterable, message -> Context.empty());
     }
 
+    /**
+     * Creates a MessageStream that provides the given {@code items} and then completes. Each item can only be consumed
+     * once.
```

Comment:
I see this javadoc is different from `fromIterable`. But is there really another behavior? Or maybe should we update the fromIterable docs as well??